src/lib/orm/orm.go

```diff
@@ -88,6 +97,12 @@ func Clone(ctx context.Context) context.Context {
 	return NewContext(ctx, orm.NewOrm())
 }
 
+// DeepClone returns new context with orm and value from parent context but no
+// linkage of parent.
+func DeepClone(ctx context.Context) context.Context {
```

Comment:
DeepClone may be inappropriate, it just copies some values, not all values.

Reply:
`CloneValue` is better?

Reply:
let's use `Copy`.